packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -31,6 +33,10 @@ const defaultStyle = {
   background: getStyle(`--vscode-panel-background`) ?? styleFallback.background,
   fontSize:
     parseInt(getStyle(`--vscode-font-size`) ?? styleFallback.fontSize) - 2,
+  foreground:
+    getStyle('--vscode-list-deemphasizedForeground') ??
```

Comment:
do I understand correctly that this is the color of edges? in that case we should be more specific with the name

Reply:
Thanks - yes, I will go with particleWidth, lineWidth and lineColor as settings.